backends/core/models.py

```diff
@@ -58,7 +67,15 @@ class Pipeline(BaseModel):
   params = relationship('Param', lazy='dynamic', order_by='asc(Param.name)')
 
   def __init__(self, name=None):
+    self.memcache_client = memcache.Client()
```

Comment:
Question: should we call `super(Pipeline, self).__init__()` here? maybe safer.

Reply:
good point. done

Reply:
done